samples/snippets/storage_restore_soft_deleted_bucket.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+
+# Copyright 2025 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import sys
+
+# [START storage_restore_soft_deleted_bucket]
+
+from google.cloud import storage
+
+
+def restore_bucket(bucket_name, bucket_generation):
+    storage_client = storage.Client()
+    bucket = storage_client.restore_bucket(bucket_name=bucket_name, generation=bucket_generation)
+    print(f"ID: {bucket.id}")
+    print(f"Name: {bucket.name}")
+    print(f"Bucket Generation: {bucket.generation}")
```

Comment:
Consider changing this to be more descriptive, similar to what's done in the java sample
```suggestion
    print(f"Soft deleted bucket {bucket.name} was restored.")
```

Reply:
Done.